src/lib/i18n/translations/keys.ts

```diff
@@ -0,0 +1,238 @@
+const keys = {
```

Comment:
`next-intl` supports type-safe messages without having to list all your keys separately: [TypeScript integration](https://next-intl.dev/docs/workflows/typescript).